pkg/cluster/delete.go

```diff
@@ -451,7 +451,7 @@ func (m *manager) Delete(ctx context.Context) error {
 		if err != nil {
 			return err
 		}
-		err = oidcbuilder.DeleteOidcFolder(ctx, env.OIDCBlobDirectoryPrefix+m.doc.ID, azBlobClient)
+		err = oidcbuilder.DeleteOidcFolder(ctx, fmt.Sprintf("%s/%s", m.subscriptionDoc.Subscription.Properties.TenantID, m.doc.ID), azBlobClient)
```

Comment:
I'd suggest making a new variable to represent the OICD issuer path and using it, instead of making the path ad-hoc.
It can be reused easily and less error-prone.

Reply:
@slawande2, I saw the change you made in response to this. I think @bitoku was saying that you could make the whole thing (i.e. `fmt.Sprintf("%s/%s", m.subscriptionDoc.Subscription.Properties.TenantID, m.doc.ID)`) a constant.

You can't use the `const` keyword though, since the value depends on the live state of the cluster manager object. I would make a utility method on the cluster manager that returns `fmt.Sprintf("%s/%s", m.subscriptionDoc.Subscription.Properties.TenantID, m.doc.ID)` and use that.

Reply:
Resolving this thread since the issue's been addressed over in https://github.com/Azure/ARO-RP/pull/3903#discussion_r1803490866